src/app/employee/employee-form/bio-form/bio-form.component.html

```diff
@@ -35,7 +35,7 @@
   </div>
   <div fxLayoutGap="20px" fxLayoutAlign="space-between center">
     <mat-form-field fxFlex="50%">
-      <mat-label>Enthnicity</mat-label>
+      <mat-label>Enthnicity *</mat-label>
```

Comment:
I think if we add `required` to the `mat-select` we'll get the asterisk automatically thanks to Material?